entries/f/fanatics.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Fanatics": {
+    "domain": "fanatics.com",
```

Do you know if the U.S. and EU/EEA sites share credentials?

```suggestion
    "domain": "fanatics.com",
    "additional-domains": [
      "fanatics.co.uk"
    ]
```

---

> Do you know if the U.S. and EU/EEA sites share credentials?

testing my login on .co.uk it is invalid.